speech/cloud-client/transcribe_async.py

```diff
@@ -71,13 +69,62 @@ def main(speech_file):
     # [END send_request]
 
 
+def transcribe_gcs(gcs_uri):
+    """Asynchronously transcribes the audio file specified by the gcs_uri."""
+    # [START authenticating_gcs]
+    # Application default credentials provided by env variable
+    # GOOGLE_APPLICATION_CREDENTIALS
+    from google.cloud import speech
+    speech_client = speech.Client()
+    # [END authenticating_gcs]
+
+    # [START construct_request_gcs]
+    # Loads the audio into memory
+    audio_sample = speech_client.sample(
+        content=None,
+        source_uri=gcs_uri,
+        encoding='FLAC',
+        sample_rate=16000)
+    # [END construct_request_gcs]
+
+    # [START send_request_gcs]
+    operation = speech_client.speech_api.async_recognize(audio_sample)
+
+    retry_count = 100
+    while retry_count > 0 and not operation.complete:
+        retry_count -= 1
+        time.sleep(2)
+        operation.poll()
```

Comment:
`_OperationFuture` has landed in GAX/GAPIC interfaces and we are slowly working it in

/cc @lukesneeringer